src/popper/scm.py

```diff
@@ -198,3 +198,11 @@ def get_branch(repo):
         return branch
 
     return get_sha(repo)
+
+
+def get_tag(repo=None):
+    """Obtains the git tag fot the currently checked our version on the given repository object
+        if possible, otherwise returns ""
```

Comment:
when a tag is not found, this is returning None rather than an empty string, which is fine, but we can update the comment to reflect that